monitoring/scripts/populate_dummy_calm_vhs.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env python
+# -*- encoding: utf-8
+
+import os
+import json
+import boto3
+import hashlib
+from download_oai_harvest import fetch_calm_records
+
+CALM_TABLE = "vhs-calm"
+CALM_BUCKET = "wellcomecollection-vhs-calm"
+
+
+def calm_records():
+    """
+    Yields calm records line by line, either from a local file downloaded
+    previously, or by downloading the records individually from the source.
+    """
+    dir_path = os.path.dirname(os.path.realpath(__file__))
+    file_path = os.path.join(dir_path, "calm_records.json")
+    print(file_path)
+
+    if os.path.exists(file_path):
+        with open(file_path) as f:
+            for record in json.load(f):
+                yield record
+    else:
```

Comment:
Ooh, looking for the file first is quite nice!

One suggestion: rather than checking `if exists(file) … else`, it's slightly better to do `try: open(file) … except FileNotFoundError`, which prevents against race conditions if the file gets deleted between your check and open.

This is admittedly unlikely, but it's one of the reasons `try … except` is a more idiomatic pattern. Use that unless you have a good reason not to.